src/components/ErrorBoundary/ErrorBoundary.jsx

```
@@ -64,6 +64,7 @@ class ErrorBoundary extends React.Component {
   render() {
     const { errorMessage, attemptedRerender } = this.state
     const { children } = this.props
+    const { t } = useTranslation()
```

ESLint

[error] React Hook "useTranslation" cannot be called in a class component. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)